src/components/ebay-menu/index.marko

```diff
@@ -25,6 +25,13 @@ $ var isCheckbox = input.type === "checkbox";
 $ var isNotCheckable = input.type !== "checkbox" && input.type !== "radio";
 $ var baseClass = input.classPrefix || "menu";
 $ var separatorCount = 0;
+$ var seperatorMap = (input.items || []).reduce((map, item, index) => {
+    if(item.separator) {
+        map[index - separatorCount] = true;
+        separatorCount++;
+    }
+    return map;
+}, {});
```

Comment:
One more nit. Sorry, last one hopefully.
Let's put this in `menuUtils`. I think what might be easiest is to create a method on menuUtils, like `getSeparatorMap`. You can move the `separatorCount` there and such.
You can then call it here using `component.getSeparatorMap(input.items)`
This way it will be only in 1 place and we don't have it copied. (I see it's copied in the other PR for fake menu.

Otherwise, it looks great!